sites/gent/ch_hpc_policies.tex

```diff
@@ -1,4 +1,85 @@
 \chapter{HPC Policies}
 \label{ch:hpc-policies}
 
-Stub chapter
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
+The conditions that apply depend on your affiliation.
+
+\section{Access for staff & academics}
+
+\subsection{Researchers and staff affiliated with Flemish university associations}
+
+\begin{itemize}
+  \item Includes externally funded researchers registered in the personnel database (FWO, SBO, VIB, IMEC, etc.)
+  \item Includes researchers from all VSC partners
+  \item Usage is free of charge
+  \item Use your account credentials at your affiliated university to request a VSC-id and connect
```

Comment:
Refer to section with more info on practical steps to do this?

Reply:
\label{ch:getting-a-hpc-account}

Reply:
I added \autoref{link} in the text, no clue how to make that change appear here.